prerequisite_changes.py

```diff
@@ -228,7 +232,53 @@ def main():
         replace("TAF/testScenarios/functionalTest/V2-API/core-command/device/GET-Positive.robot",
                 "== 6",
                 "== ${length}")
+    if service == "4_ping_response_time" or service=="performance-metrics-collection":
+        # Added dependent library in test robot file
+        add_data("TAF/testScenarios/performanceTest/performance-metrics-collection/4_ping_response_time/ping_response_time.robot", 
+                 "Resource        TAF/testCaseModules/keywords/common/commonKeywords.robot\n",
+                 "Variables         TAF/config/performance-metrics/configuration.py\n")
+        # Remove service ping as service is not available in swagger but was available in docker container
```

Comment:
ping api is already available in our edgex also for all the services, we have that API exposed in Swagger page.

Reply:
I want to discuss offline also.

Reply:
We are already doing ping test for all the services in our smoke test file.

https://github.com/etn-ccis/edge-linux-test-edgex-pytest/blob/e2c90729943b6b57cd4298927e85156fc3cea413/px_red/tests/edgex/test_edgex_smoke.py#L114